sdks/python/generate_pydoc.sh

```diff
@@ -59,8 +62,11 @@ EOF
 
 # Build the documentation using sphinx
 python $(type -p sphinx-build) -q target/docs/source target/docs/_build -c target/docs/source \
-    -w "/tmp/sphinx-build.warnings.log"
+    -w "target/docs/sphinx-build.warnings.log"
 
 # Message is useful only when this script is run locally.  In a remote
 # test environment, this path will be removed when the test completes.
 echo "Browse to file://$PWD/target/docs/_build/index.html"
+
+# Fail if there are errors in docs
+! grep -q "ERROR:" target/docs/sphinx-build.warnings.log
```

Comment:
Sounds good, we can keep tracking them on the issue. This is a good change by itself.